src/main/java/co/omise/Example.java

```diff
@@ -254,7 +253,8 @@ void destroyTransfer() throws IOException, OmiseException, ClientException {
     void listTransfers() throws IOException, OmiseException, ClientException {
         Request<ScopedList<Transfer>> request = new Transfer.ListRequestBuilder()
                 .build();
-        ScopedList<Transfer> transfers = client().sendRequest(request, new TypeReference<ScopedList<Transfer>>() {});
+        ScopedList<Transfer> transfers = client().sendRequest(request, new TypeReference<ScopedList<Transfer>>() {
+        });
```

Comment:
Shall we keep this in the same line?

Reply:
Autoformat keeps bringing it back down :/